cfg/builder/builder_entry.cc

```diff
@@ -29,12 +29,11 @@ unique_ptr<CFG> CFGBuilder::buildFor(core::Context ctx, ast::MethodDef &md) {
 
         auto selfClaz = md.symbol.data(ctx)->rebind();
```

Comment:
`rebind` is already a `ClassOrModuleRef` with my change, so we don't need to call enclosingClass on it.